File: gdscript/src/main/java/gdscript/psi/GdPrimaryEx.java
package gdscript.psi;

import org.jetbrains.annotations.*;

public interface GdPrimaryEx extends GdExpr {

  @Nullable
  GdArrayDecl getArrayDecl();

  @Nullable
  GdDictDecl getDictDecl();

  @Nullable
  GdExpr getExpr();

  @Nullable
  GdNodePath getNodePath();

}

File: gdscript/src/main/java/gdscript/psi/GdPrimaryEx.kt
package gdscript.psi;

import org.jetbrains.annotations.Nullable;

public interface GdPrimaryEx extends GdExpr {

  @Nullable
  GdArrayDecl getArrayDecl();

  @Nullable
  GdDictDecl getDictDecl();

  @Nullable
  GdExpr getExpr();

  @Nullable
  GdNodePath getNodePath();

}
